rollup.config.js

```diff
@@ -26,7 +26,6 @@ export default {
   input,
   plugins: [
     litcss({
-      include: '/**/*.css',
```

Comment:
It matches css files by default, using regex. It seems that regex works cross-platform and glob is POSIX only.